manifests/g/gaubert/gmvault/1.9.1/gaubert.gmvault.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: gaubert.gmvault
+PackageVersion: 1.9.1

```

Comment:
```suggestion
PackageVersion: 1.5-beta
```

Reply:
Thanks for your help

Reply:
I had noticed this as well but decided to take the version on the website of the download. I will keep this in mind for future PRs.

Reply:
Yeeeah WinGet looks at the Version which shows up in the Control Panel, so for now use that.